server/api/login.js

```diff
@@ -0,0 +1,78 @@
+const router = require('express').Router();
+const { User, Session } = require('../db').models;
+
+const A_WEEK_IN_SECONDS = 60 * 60 * 24 * 7;
```

Comment:
Might want to multiply by 1000 some cookies seem to use MS instead of S for some reason?